### DISCUSSION TITLE
Can a store have multiple hierarchies or is a store only associated with exactly one heirarchy?

### DISCUSSION THREAD

**Discussion by zoj613** [zarr-developers/zarr-specs#297] [Q&A] [answered]
This isn't clear to me from reading the v3 spec. If one creates a store, can one store multiple hierarchies in a single store where each hierarchy has it's own "root"? Or is this not allowed?

**Comment by jbms** [comment, marked as answer]
> This isn't clear to me from reading the v3 spec. If one creates a store, can one store multiple hierarchies in a single store where each hierarchy has it's own "root"? Or is this not allowed?

In the abstract this question may not be extremely meaningful, because you can always just define the store to be a sub- directory/key-prefix within another store.

In particular, though, you can indeed have multiple zarr hierarchies within one S3 bucket or one zip file or one local filesystem.

**Reply by zoj613** [reply]
Thanks a lot for the quick reply. That clarifies a few things for me.
